host/class/cdc/usb_host_cdc_acm/test_app/main/test_app_main.c

```diff
@@ -35,23 +37,7 @@ void app_main(void)
     printf("|______/ /_______  / |______  /  |__|  \\___  >____  > |__|  \r\n");
     printf("                 \\/         \\/             \\/     \\/        \r\n");
 
-    UNITY_BEGIN();
+    unity_utils_setup_heap_record(80);
+    unity_utils_set_leak_level(530);
```

Comment:
With moving the memory leak start before the `unity_run_menu` we measure the memory leak for all tests runs together.

It would be better if we evaluate the memory leaks per test like this
``` c
void setUp(void)
{
    unity_utils_set_leak_level(SOME_VALUE);
    unity_utils_record_free_mem();
}

void tearDown(void)
{
    unity_utils_evaluate_leaks();
}
```

Reply:
Good point, thx for checking.

I updated all the `test_app_main.c` accordingly to our test apps in esp-idf

``` c
void setUp(void)
{
    unity_utils_record_free_mem();
}

void tearDown(void)
{
    unity_utils_evaluate_leaks();
}

void app_main(void)
{
    ....
    unity_utils_setup_heap_record(SOME_VAL);
    unity_utils_set_leak_level(SOME_VAL);
    unity_run_menu();
}